planner/core/find_best_task.go

```diff
@@ -1023,6 +1023,8 @@ func (ds *DataSource) getOriginalPhysicalTableScan(prop *property.PhysicalProper
 	}.Init(ds.ctx, ds.blockOffset)
 	if ds.preferStoreType&preferTiFlash != 0 {
 		ts.StoreType = kv.TiFlash
+		ts.filterCondition = append(ts.filterCondition, ts.AccessCondition...)
```

Comment:
Are these 2 lines necessary?

Reply:
Oh... That the code for another PR. I pushed the wrong branch.